views/components/table.html

```diff
@@ -0,0 +1,25 @@
+{% macro table(tableData) %}
+<article class="base--article" itemscope itemtype="http://schema.org/Table">
+  <header class="base--header">
+    <h1 class="base--h1" itemprop="name">{{tableData.name}}</h1>
```

Comment:
The h1 should have been dropped in favor of `caption`. A `caption` is the title of a table.
http://html5doctor.com/element-index/#caption

As this is a simple table component, it also shouldn't be wrapped in an `article`. `table` is a self-contained semantic object